### DISCUSSION TITLE
Support for custom database ?

### DISCUSSION THREAD

**Discussion by NargiT** [renovatebot/renovate#16523] [Request Help] [answered]
Hello,

Is there any support for custom database ? My objective is to reused an existing REST api to allow renovate to query it as a new datasource. Or at least give me pointers to implement a custom module for datasources.

**Comment by rarkins** [comment, marked as answer]
Related: https://github.com/renovatebot/renovate/issues/10622

The idea of this one is if you can add your own endpoint to provide results in a format Renovate understands.

Alternatively we are also considering the concept of a generic datasource which could be similar to #15193
